external/test-runtime/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "xunit.console.netcore": "1.0.3-prerelease-00921-01",
+    "xunit.runner.console": "2.2.0-beta2-build3300",
```

Comment:
I think we have the test-runtime to restore test dependencies related to the tests and not the framework. I am not feeling strongly either way but what have here is clear split what the test need and what the framework need.